### ISSUE TITLE
Simple smart answers - check answers link readable

### ISSUE THREAD

**Issue by injms** [alphagov/frontend#2214]
Simple smart answers play back the answers given, similar to the [check answers](https://design-system.service.gov.uk/patterns/check-answers/) pattern shown in the GOV.UK Design System.

---

Example:
![](https://user-images.githubusercontent.com/861310/72986248-b44a7000-3ddf-11ea-88ad-c8a9eada0306.png)

---

The link in table - 'Change' - is read aloud by screenreaders as 'Change this answer'. This means that the link lacks context. Assistive tech, such as screenreaders, can pull out all of the links in a page to allow a user to skim through them. 

In the above example there'd be two 'Change this answer' present in that list, neither of which make sense on their own. I think that this could be improved by adding context to the link.

I think this will need collaboration with some wordsmiths to make sure that the link isn't too verbose and can still make sense when looked at on its own.

I appreciate that this could get complicated, especially as the answer a question can rely on context from previous answers. Since this form is playing back information to a user that they have entered, I wonder if we can assume a user has a reasonable level of familiarity with the information? 

Since the links don't make sense on their own, any improvement to this will help users who make use of assistive technologies.

**Comment by danacotoran** [comment]
I believe this has already been fixed through [the use of the summary list component](https://github.com/alphagov/frontend/commit/f1e5a2b222755764ad87ff377f6ca480d3f3d668#diff-e22a87beb13570f41c6b5011887fa6f6d852a856adfe7d760abc8bb74385e45c), which adds hidden text to the action links by default (so a screenreader would read out the first link in the example in the description as **"Change 1. What's your situation?"** which should be enough context).

Therefore, I shall take the liberty of closing this! 👀
